components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -779,7 +771,7 @@ struct NetworkInfo {
   string symbol_name;
   int32 decimals;
   CoinType coin;
-  NetworkInfoData? data;
+  bool is_eip1559; // only for ETH.
```

Comment:
Let's make code needing this be less verbose. In case we really need to support much per coin data I suggest adding optional per-coin structs instead of union.

Reply:
cc @nuo-xu and @StephenHeaps in case iOS needs to update.